README.md

```diff
@@ -1 +1,47 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse
+
+This project demostrates usage of linters in html and css
+
+# Built With
+-Html and css
+
+# Getting Started
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
+
+To get a local copy up and running follow these simple example steps.
+
+# Prerequisites
+-text editor
+-nodejs installed on your local machine
+
+# Setup
+-Clone the repository
+-Open the project in your favourite text editor
+
+# Install
+-Run npm install to install project dependencies
+
+# Authors
+-GitHub: https://github.com/Daniel-Kigozi,
+-Twitter: @DanielKigozi15,
+-LinkedIn: https://www.linkedin.com/in/daniel-kigozi-375182208/
+
+handshake Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+# Show your support
+Give a star if you like this project!
+
+# Acknowledgments
+
+Hat tip to anyone whose code was used
+Inspiration
+etc
+
+# License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
**[OPTIONAL]:** This is just a set of instructions to let you know where to get a license for your project. I would recommend removing it as this will make your README cleaner :sparkles: :100: